src/main/java/us/dot/its/jpo/ode/aws/depositor/AwsDepositor.java

```diff
@@ -158,14 +163,22 @@ public void run(String[] args) throws Exception {
 		boolean depositToS3 = false;
 		AmazonS3 s3 = null;
 		AmazonKinesisFirehoseAsync firehose = null;
+		Storage gcsStorage = null;
+
 		if (destination != null && destination.equals("s3")) {
 			depositToS3 = true;
 			s3 = createS3Client(awsRegion);
 
-		} else {
+		} else if (destination != null && destination.equals("firehose")) {
 			firehose = buildFirehoseClient(awsRegion);
+		} else if (destination != null && destination.equals("gcs")) {
+			gcsStorage = StorageOptions.getDefaultInstance().getService();
+		} else {
+			logger.error("Invalid destination: " + destination);
+			System.exit(1);
 		}
```

Comment:
This logic could be simplified by pulling out the null check into a separate if statement.

Reply:
Pulled the null check into a separate if statement